Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'gds-api-adapters', '40.1.0'
 gem 'airbrake', '4.0.0'
 gem 'logstasher', '0.6.2'
 
-gem 'govuk_navigation_helpers', '~> 3.1'
+gem 'govuk_navigation_helpers', '~> 3.1.1'
```

Comment:
This isn't necessary - unless your intent was explicitly to ensure we only get bug fixes on `bundle update`?Despite recent versioning shenanigans on that gem, IMHO we should continue to try and conform to SemVer and have this line declare the dependency at the minor patch level.